### DISCUSSION TITLE
External download manager,3rd party certificate

### DISCUSSION THREAD

**Discussion by iamthelara** [bromite/bromite#1205] [Ideas]
hello, dev, I download lot of files for my office work so i need a download manager. Recently i read about your privacy concern about adding ext download manager feature. I talked to 1DM dev about this and asked n dev gave me this: 1DM integration code if the dev needs

https://www.apps2sd.info/idmp/Util1DM.java

And I can't use https filtering in Adguard So any help?

**Comment by uazo** [comment]
> 1DM integration code if the dev needs

It is not possible. bromite will never ask the user to install anything.
try using the share link function

> And I can't use https filtering in Adguard So any help?

I think you need to check `#allow-user-certificates` flag

**Comment by iamthelara** [comment]
Got it. But can't we get external download manager feature ever?

**Reply by uazo** [reply]
> But can't we get external download manager feature ever?

maybe with https://github.com/bromite/bromite/issues/792

**Comment by iamthelara** [comment]
I can't find user script settings. Any help?

**Reply by uazo** [reply]
you have to wait a little longer. the feature is still in development.

**Comment by iamthelara** [comment]
Any flag or setting to disable recent/most visited sites?

And can we use google transalor in userscripts?

**Reply by Issam28** [reply]
I second this. Having the option to disable most visited sites would be helpful.
